src/NServiceBus.Metrics/Reporters/NServiceBusMetricReport.cs

```diff
@@ -35,6 +39,8 @@ public void RunReport(MetricsData metricsData, Func<HealthStatus> healthStatus,
     {
         var stringBody = $@"{{""Data"" : {JsonBuilderV2.BuildJson(metricsData)}}}";
         var body = Encoding.UTF8.GetBytes(stringBody);
+
+        headers[Headers.OriginatingEndpoint] = metricsData.Context; // assumption that it will be always the endpoint name
```

Comment:
We thought about that, the implications would be the MetricsOptions needs to implement ExposeSettings or the builder func needs to expose the read-only settings which we figured is not really required yet. The verification is contained in the tests, so whenever someone changes the context to be something else the test breaks.